scripts/functions.js

```diff
@@ -0,0 +1,500 @@
+// functions.js defines functions that will be called later
+
+
+// global variable for the path to the historical districts data file
+var districtsFile = 'data/qrySumStatsAllDistAllYears.csv';
+// data structure to hold state for the chart; the actual data will be attached on load
+// see https://github.com/d3/d3-format#locale_format for tick format strings
+var chartData = {
+	svgID: 'chart',
+	visible: true,
```

Comment:
If you want the chart to be folded away when the page first loads, just set this to `false`.

Reply:
This is really helpful. Is the var districtsFile the only place where csv needs to be specified? So when we swap in updated data, if we choose to change file name, we just change it here and charts will still work?

Reply:
That's correct, as long as the variable names are consistent.